test/staleIfError.js

```diff
@@ -461,4 +467,130 @@ describe('Stale-If-Error', () => {
       assert.fail('Expected to throw');
     });
   });
+
+  describe('Cache keys', () => {
+    it('keys cache entries by method and url', async () => {
+      const cache = createCache();
+
+      api.get('/').reply(200, defaultResponse.body, defaultHeaders);
+
+      const maxAge = 60000;
+      const staleIfError = 7200000;
+      const expiry = Date.now() + maxAge + staleIfError;
+
+      await createCacheClient(cache)
+        .get('http://www.example.com/')
+        .asResponse();
+
+      const cached = await cache.get({
+        segment: `http-transport:${VERSION}:stale`,
+        id: 'GET:http://www.example.com/'
+      });
+
+      const actualExpiry = cached.ttl + cached.stored;
+      const differenceInExpires = actualExpiry - expiry;
+
+      assert.deepEqual(cached.item.body, defaultResponse.body);
+      assert(differenceInExpires < 1000);
```

Comment:
Does this assert correctly? Do you not need `.to.be.true`?
Could you flip the assertion and make sure the test fails.

Reply:
`assert()` is an alias of `assert.ok()`. If the comparison inside is not true it will fail.

Reply:
OK. Maybe it's `expect()` where the problem happens.

Reply:
Yeah, I believe `expect()` behaves differently.